heron/common/src/cpp/config/topology-config-helper.cpp

```diff
@@ -197,5 +197,60 @@ sp_int64 TopologyConfigHelper::GetContainerRamRequested(const proto::api::Topolo
       (total_parallelism / nstmgrs) + (total_parallelism % nstmgrs);
   return max_components_per_container * 1073741824l;
 }
+
+bool TopologyConfigHelper::IsTopologyStateful(const proto::api::Topology& _topology) {
```

Comment:
The 3 bool methods have identical implementations except for the `TopologyConfigVars` object checked. Can you have each method instead delegate to an internal helper that centralizes this logic that takes a `TopologyConfigVars` object?